Dockerfile

```diff
@@ -34,7 +34,10 @@ ENV SNMP_HOST="localhost"
 ENV SNMP_RETRIES=5
 ENV SNMP_TIMEOUT=1
 ENV ALERT_OID_LABEL="oid"
+ENV TRAP_OID_PREFIX="1.3.6.1.4.1.50495.15"
+ENV TRAP_DEFAULT_OID="1.3.6.1.4.1.50495.15.1.2.1"
```

Comment:
That's pretty interesting. I hadn't actually though to lookup the OIDs, and when I was looking at my output, it seemed like it converted it to an "enterprise" value for the prefix, so I thought it was already pretty generic. Seems that's not really the case. It might be worth doing some changes here to get this setup nicer.

Also the changes I'm proposing here and in service-telemetry-operator allow us to perform per-alert overrides, so someday it might be worth finding appropriate OIDs and documenting them so the alerts defined in our alerts.yaml provide an appropriate OID for the type of alarm they are firing for.

Reply:
This is some really good stuff, and I think deserves a little bit of investigation and perhaps clean up and documentation that would help make some of this functionality a bit less clunky and possibly useful for some use cases.

I'm going to track some investigation and perhaps suggested changes at https://issues.redhat.com/browse/STF-1257.

Reply:
Ah I found the `15` I think you were asking about. Someone else asked on the upstream project about the MIB so they could import it, and the response was to look here: https://github.com/SUSE/prometheus-webhook-snmp/blob/master/PROMETHEUS-ALERT-CEPH-MIB.txt

Looking at that file on lines 10 and 11 you can see the definition of 15:

```
ceph       OBJECT IDENTIFIER ::= { enterprises 50495 }
prometheus OBJECT IDENTIFIER ::= { ceph 15 }
```

If you keep following the OID path you get the rest of the information that's been defined by default. I think this means there is no need to change the defaults currently. I'm going to follow this path in the MIB and further define what the default value is, and verify that is indeed the default set we want.

From there I'm going to work on updating our concept module for SNMP stuff in the documentation repo. Once this change set merges down in combination with the service-telemetry-operator updates, administrators will be able to override the default `oid` via label in the alert definition like so:

```yaml
apiVersion: monitoring.coreos.com/v1
kind: PrometheusRule
metadata:
  labels:
    prometheus: stf-default
    role: alert-rules
  name: prometheus-alarm-rules
  namespace: service-telemetry
spec:
  groups:
  - name: ...
    rules:
    - alert: ...
      annotations:
        action: ...
        description: ...
      expr: sg_total_collectd_msg_received_count > 1
      for: 10s
      labels:
        cluster: TEST
        severity: warning
        oid: 1.3.6.1.4.1.50495.15.1.2.42
```